USAGE.md

```diff
@@ -0,0 +1,48 @@
+Using UniMath
+=============
+
+
+Browsing and editing a file
+---------------------------
+Once you have [installed](./INSTALL.md) UniMath, you can start browsing and editing the source files.
+There are several programs to interactively edit and step through the files, among which
+are [CoqIDE](https://coq.inria.fr/refman/practical-tools/coqide.html) and [ProofGeneral](https://proofgeneral.github.io/).
+Here, we focus on ProofGeneral.
+
+During the [installation procedure](./INSTALL.md) you have set up ProofGeneral on your computer.
+ProofGeneral is an add-on to the text editor Emacs, adding buttons, menus, and keyboard shortcuts
+to interact with Coq, the proof assistant that UniMath relies on.
+
+Upon opening a Coq file (i.e., a file with ending *.v) in Emacs, the ProofGeneral add-on is automatically
+loaded by Emacs.
+Andrej Bauer has a [screencast](https://www.youtube.com/watch?v=l6zqLJQCnzo) on using ProofGeneral
+(more screencasts are listed on his [website](http://math.andrej.com/2011/02/22/video-tutorials-for-the-coq-proof-assistant/)).
+Note that for Bauer's specific example to work in UniMath, you need to insert the line
```

Comment:
Maybe say that ProofGeneral doesn't load the prelude only if you are stepping through I file located in our UniMath directory tree.  It's also possible to use ProofGeneral outside that tree, if Coq is installed on the system.